api/api/services/internal_tasks/integration_service.py

```diff
@@ -267,7 +268,7 @@ async def _find_relevant_run_and_agent(
             # There is a change we are capturing a pre-existing, unnamed agent run here, but this is pretty rare, and low impact
             # Especially since the proposed agent name we'll compute will allow the user to understand where the captured run is coming
             # And this will hopefully convince them to name the pre-existing, unnamed agent.
-            if agent.name == DEFAULT_AGENT_ID:
+            if agent.name.lower() == DEFAULT_AGENT_ID.lower():
```

Comment:
Why not using the agent id directly?

Reply:
right ! Updated.https://github.com/WorkflowAI/WorkflowAI/pull/286/commits/047c1112394f132e33f16cd3c04c164cc0df1029